src/components/Garden/Footer.js

```diff
@@ -107,14 +112,19 @@ function Footer() {
   )
 }
 
-function FixedFooter({ token }) {
+function FixedFooter({ token, gardenAddress }) {
   const theme = useTheme()
+  const history = useHistory()
   const { account } = useWallet()
   const { layoutName } = useLayout()
   const [createProposalModalVisible, setCreateProposalModalVisible] = useState(
     false
   )
 
+  const handleOnGoToCovenant = useCallback(() => {
+    history.push(`/garden/${gardenAddress}/covenant`)
```

Comment:
We can use `buildGardenPath` so that it´s not necessary to pass the gardenAddress as prop.